docs/docker/Dockerfile

```diff
@@ -4,14 +4,16 @@ ENV PYTHONUNBUFFERED 1
 
 ENV OPENPILOT_PATH /home/batman/openpilot/
 ENV PYTHONPATH ${OPENPILOT_PATH}:${PYTHONPATH}
+ENV POETRY_VIRUALENVS_CREATE false
 
 RUN mkdir -p ${OPENPILOT_PATH}
 WORKDIR ${OPENPILOT_PATH}
 
-COPY Pipfile Pipfile.lock $OPENPILOT_PATH
-RUN pip install --no-cache-dir pipenv==2021.5.29 pip==21.3.1 && \
-    pipenv install --system --deploy --dev --clear && \
-    pip uninstall -y pipenv
+COPY pyproject.toml poetry.lock $OPENPILOT_PATH
+RUN pip install --no-cache-dir poetry==1.2.1 pip==22.2.2 && \
+    poetry install --no-ansi --no-interaction && \
+    pip uninstall -y poetry && \
+    rm -rf /root/.cache
```

Comment:
hmm, is this even necessary? can we just remove instead? this should already be done in openpilot-base